### PR TITLE
feat(blocks): autofocus new block inserted from block hub

### PR DESCRIPTION
**Pull request by thorseraq**
https://user-images.githubusercontent.com/20554850/215693082-7e4db235-2838-41b8-824f-d2b0160a0e21.mov

### CODE REVIEW

File: packages/store/src/workspace/page.ts [hunk 1]
```diff
@@ -471,28 +471,36 @@ export class Page extends Space<PageData> {
...
           type: blockProps.type,
         },
         targetParentModel.id,
         top ? targetIdx : targetIdx + 1
       );
-      // }
+    });
+    if (!autofocus) return;
+    // borrowed from asyncFocusRichText() in @blocksuite/blocks
+    requestAnimationFrame(() => {
```
Comment (doodlewind) [inline packages/store/src/workspace/page.ts:500]: We shouldn't do this kind of UI mutation in the `store` package. Can you try to move the `insertBlock` method to the `blocks` package?

Comment (doodlewind) [reply packages/store/src/workspace/page.ts:500]: Let me try to refactor here a bit.

Comment (thorseraq) [reply packages/store/src/workspace/page.ts]: Thanks！！I will take care next time

**Review by doodlewind — APPROVED** [review]
LGTM 👻

**Pull request merged**